### PR TITLE
don't use an error exit code if all the messages are info.

### PR DESCRIPTION
**Pull request by dnl**
Hi, 

I'm having this issue with exit code on info messages.

The doc implies that using the `--exit-code` flag, will only fail for errors and warnings.
```
--exit-code	Exits with non-zero code if there were any warnings/errors.
```

It was also returning non-zero exit code if all the messages are `info` level messages (in my case, pronto-rubocop 'refactor' severity)

Thanks for your time. 

Dan.

### CODE REVIEW

File: lib/pronto/cli.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def run(path = nil)
       messages = Dir.chdir(repo_workdir) do
         ::Pronto.run(commit, '.', formatters, path)
       end
-      exit(messages.count) if options[:'exit-code']
+      exit(1) if options[:'exit-code'] && messages.any? { |m| m.level != :info }
```
Comment (mmozuras) [inline lib/pronto/cli.rb]: Why did you change `messages.count` into `1`? 😄 
Let's tackle this separately.

Comment (mmozuras) [inline lib/pronto/cli.rb]: @doomspork could you give a source that would back up this statement? 😄 

From what I understand, there's only one hard rule: `0` means `success`. Other exit codes can be defined by each application individually. Reference: http://stackoverflow.com/a/1101969

**Comment by dnl** [comment]
I've updated this to output the exit code as the number of errors

**Comment by mmozuras** [comment]
@dnl thanks! 🙇 👍
